0102-Binary-Tree-Level-Order-Traversal.py

```diff
@@ -0,0 +1,33 @@
+'''
+Given a binary tree, return the level order traversal of its nodes' values. (ie, from left to right, level by level).
+
+For example:
+Given binary tree [3,9,20,null,null,15,7],
+    3
+   / \
+  9  20
+    /  \
+   15   7
+return its level order traversal as:
+[
+  [3],
+  [9,20],
+  [15,7]
+]
+'''
+class Solution:
+    def levelOrder(self, root: TreeNode) -> List[List[int]]:
+        if root is None:
+            return root
+        queue, res = [root], []
+        while len(queue) > 0:
+            temp = []
+            for _ in range(len(queue)):
```

Comment:
what's the use of range here?

Reply:
It hadn't worked when I did `for _ in queue:`, if that is what you mean to say

Reply:
try something else

Reply:
okay, thanks